packages/toggle/src/styles.ts

```diff
@@ -0,0 +1,289 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme, createUniqueClassName } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { focusRing, hoverRing } from '@leafygreen-ui/tokens';
+import { Size } from './types';
+
+export const toggleButtonClassName = createUniqueClassName('toggle-button');
+
+export const buttonSelectors = {
+  checked: `.${toggleButtonClassName}[aria-checked="true"]`,
+  unchecked: `.${toggleButtonClassName}[aria-checked="false"]`,
+  disabled: `.${toggleButtonClassName}:disabled`,
+};
+
+export const sliderSelector = {
+  checked: `${buttonSelectors.checked} > &`,
+  unchecked: `${buttonSelectors.unchecked} > &`,
+  disabled: `${buttonSelectors.disabled} > &`,
+};
+
+export const checkmarkSelector = {
+  checked: `${buttonSelectors.checked}:not(:disabled) &`,
+  unchecked: `${buttonSelectors.unchecked}:not(:disabled) &`,
+  disabledChecked: `${buttonSelectors.checked}:disabled &`,
+  disabledUnchecked: `${buttonSelectors.unchecked}:disabled &`,
+};
```

Comment:
🔥

Reply:
i didn't actually add this, I only moved all the styles over to the new style file